### PR TITLE
[HELM] use minio on teresa chart

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/314)

### CODE REVIEW

File: helm/chart/teresa/values.yaml [hunk 1]
```diff
@@ -30,3 +30,6 @@ build:
     cpu: 500m
     memory: 1024Mi
 debug: false
+useMinio: false
+minio:
+  serviceType: ClusterIP
```
Comment (drgarcia1986) [inline helm/chart/teresa/values.yaml]: why not ELB? are you thinking about minikube?

Comment (yagonobre) [reply helm/chart/teresa/values.yaml]: I guess that the `minio` will only be used by `teresa` then I don't need an ELB. Makes sense?

Comment (drgarcia1986) [reply helm/chart/teresa/values.yaml]: I'm not sure about that. (Don't forget you need to create the bucket before use with Teresa)

Comment (yagonobre) [reply helm/chart/teresa/values.yaml]: Now this create the bucket

**Comment by yagonobre** [comment]
waiting https://github.com/kubernetes/charts/pull/2061

**Comment by drgarcia1986** [comment]
What about this PR ?

**Comment by yagonobre** [comment]
@drgarcia1986 needs rebase but LGTM

**Comment by yagonobre** [comment]
@drgarcia1986 done

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 rebased

**Pull request merged**